tests/test_toys.py

```diff
@@ -3,12 +3,12 @@
 
 
 def test_smoketest_toys(backend):
-    tb, _ = backend
+    tb, _ = pyhf.get_backend()
```

Comment:
this should not have been changed. We have to inject the backend this way for testing purposes (if the tests are in weird orders, the `pyhf.get_backend()` isn't properly reset for tests).